documentation/API_documentation/NumberVerification-Readiness-Checklist.md

```diff
@@ -0,0 +1,27 @@
+# API Readiness Checklist
+
+Checklist for number_verification 1.0.0-rc.1 in release r1.1
+
+| Nr | API release assets  | alpha | release-candidate |  public-release<br>initial | public-release<br> stable | Status | Comments |
+|----|----------------------------------------------|:-----:|:-----------------:|:-------:|:------:|:----:|:----:|
+|  1 | API definition                               |   M   |         M         |    M    |    M   |   Y   | [link](/code/API_definitions/number_verification.yaml) |
+|  2 | Design guidelines from Commonalities applied |   O   |         M         |    M    |    M   |  Y    |      |
+|  3 | Guidelines from ICM applied                  |   O   |         M         |    M    |    M   |   Y   |      |
+|  4 | API versioning convention applied            |   M   |         M         |    M    |    M   |   Y   |      |
+|  5 | API documentation                            |   M   |         M         |    M    |    M   |   Y   | Embed documentation into API spec - [link](/code/API_definitions/number_verification.yaml)  |
+|  6 | User stories                                 |   O   |         O         |    O    |    M   |   tbd   | [PR #129](https://github.com/camaraproject/NumberVerification/pull/129/files) |
+|  7 | Basic API test cases & documentation         |   O   |         M         |    M    |    M   |   tbd   | [PR #124](https://github.com/camaraproject/NumberVerification/pull/124/files) |
+|  8 | Enhanced API test cases & documentation      |   O   |         O         |    O    |    M   |   tbd  | [PR #124](https://github.com/camaraproject/NumberVerification/pull/124/files) |
+|  9 | Test result statement                        |   O   |         O         |    O    |    M   |   N   | link |

```

Comment:
Test result statement is mandatory for a stable release - at least it has to be discussed how this can be achieved until M5.

```suggestion
|  9 | Test result statement                        |   O   |         O         |    O    |    M   |   tbd   | link |

```